src/twisted/test/test_udp.py

```diff
@@ -307,6 +307,66 @@ def cbFinished(ignored):
         d.addCallback(cbFinished)
         return d
 
+    def test_server_read_failure(self):
```

Comment:
Twisted coding standard for method names has this as `test_serverReadFailure`; I'm sorry the docs don't make this clearer and more discoverable.